static/index.html

```diff
@@ -386,6 +386,9 @@
         <section id="addon-config-settings" class="hidden settings-subsection">
         </section>
         <section id="addon-discovery-settings" class="hidden settings-subsection">
+          <div class="section-title-container">
+            Search: <input id="discover-addons-search"></input>
```

Comment:
The "Search" string should probably just be a placeholder for the input, and it needs to be pulled into the Fluent translation. So, the HTML should be this:

```html
<input id="discover-addons-search" data-l10n-id="addon-search"></input>
```

Then, in `static/fluent/en-US/main.ftl`, in the "Add-on Settings" section, you should have:

```
addon-search =
    .placeholder = Search
```

Reply:
Done